modules/openideal_slideshow/templates/slideshow.html.twig

```diff
@@ -1,5 +1,77 @@
+{# The swiper slideshow. #}
 <div class="openideal-slideshow">
-    {% for item in items %}
-        <img src="{{ item.uri.value | image_style('slideshow_large') }}" alt="{{ item._referringItem.alt }}">
-    {% endfor %}
+    <ul class="swiper-wrapper my-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+        {% for key, item in items %}
+            <li id="{{ key + 1 }}" class="swiper-slide" itemprop="associatedMedia" itemscope
+                itemtype="http://schema.org/ImageObject">
+                {# For the photoswipe use default image without styles. #}
+                <a id="{{ key == 0 ? 'first' : '' }}" title="click to zoom-in" href="{{ file_url(item.uri.value) }}"
+                   itemprop="contentUrl" data-size="1200x600">
+                    <img itemprop="thumbnail" src="{{ item.uri.value | image_style('slideshow_large') }}"
```

Comment:
Should we add item attributes to the markup?
See default core `ImageFormatter.php` for reference.